src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/RejectionException.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.ErrorCode;
+
+public abstract class RejectionException extends RuntimeException {
```

Comment:
This is a suggestion only. I probably proposed this name, but I'm not sure it's descriptive enough. Maybe `EnvelopeRejectionException`?